drivers/fastboot/Kconfig

```diff
@@ -218,6 +218,12 @@ config FASTBOOT_CMD_OEM_BOOTBUS
 	  Add support for the "oem bootbus" command from a client. This set
 	  the mmc boot configuration for the selecting eMMC device.
 
+config FASTBOOT_CMD_OEM_UCMD
+	bool "Enable the 'oem ucmd' command"
```

Comment:
After I added this as a subscriber patch, I wondered if this shouldn't be default n.

@ricardosalveti ?

Reply:
^^ @ricardosalveti